packages/shared/src/lib/click.ts

```diff
@@ -0,0 +1,6 @@
+export const combinedClicks = (func) => {
+  return {
+    onMouseUp: (event) => event.button === 1 && func,
```

Comment:
@idoshamun My concern is with this one and options where we use (e) => preventdefault passthrough.

Reply:
You forgot to call the function. Also I don't see the input types.
Not sure about your comment, can you elaborate please?

```suggestion
    onMouseUp: (event) => event.button === 1 && func(event),
```

Reply:
Well the issue is often the function takes multiple arguments.
So onClick=(e, something, fuu) => callFunc(e, something, fuu)

Reply:
If you want, you can use also (event, ...args) => event.button === 1 && func(event, ...args)